lib/modules/manager/regex/utils.ts

```diff
@@ -97,3 +97,11 @@ export function mergeExtractionTemplate(
     replaceString: addition.replaceString ?? base.replaceString,
   };
 }
+
+export function isValidDependency({
+  depName,
+  currentValue,
+}: PackageDependency): boolean {
+  // check if all the fields are set
+  return Boolean(depName) && Boolean(currentValue);
```

Comment:
use `is.string` instead, probably `isNonEmptyOrWhitespaveString`

@HonkingGoose we should add a best practices to not use `Boolean` ans use `is` functions instead

Reply:
See PR:

- https://github.com/renovatebot/renovate/pull/16573